src/goto-instrument/code_contracts.cpp

```diff
@@ -563,7 +568,11 @@ void code_contractst::instrument_assign_statement(
   alias_assertion.add(goto_programt::make_assertion(
     assigns_clause.alias_expression(lhs),
     instruction_iterator->source_location));
-
+  std::string source_expr_string;
+  (get_language_from_identifier(ns, lhs.id()))
+    ->from_expr(lhs, source_expr_string, ns);
```

Comment:
These 3 lines look like we are inlining the definition of https://github.com/diffblue/cbmc/blob/48893287099cb5780302fe9dc415eb6888354fd6/src/langapi/language_util.cpp#L20-L31

Could you please explain what is the issue with doing the following (after `#include`ing the appropriate headers):
```c
    "Check that " + from_expr(ns, lhs.id(), lhs) + " is assignable"
```

Reply:
Perfect! Implemented as suggested.